### DISCUSSION TITLE
"Enter password pop up" every time even though no password required

### DISCUSSION THREAD

**Discussion by PovilasC** [dbeaver/dbeaver#13682] [Q&A]
<img width="557" alt="Screenshot 2021-07-20 at 16 33 44" src="https://user-images.githubusercontent.com/2620949/126458640-fea8fd01-6878-48b4-906b-2a53a394666b.png">

#### System information: 
- Operating system (distribution) and version: macOS Big Sur 11.4
- DBeaver version: 21.1.3.202107181810

#### Connection specification:
- Database name and version: postgres
- Driver name
- Do you use tunnels or proxies (SSH, SOCKS, etc)? 

#### Describe the problem you're observing:
I use DBeaver to connect to my remote PostgreSQL DB hosted on Linux (AWS Lightsail) through self-signed SSL certificate. The problem is that every time I open DBeaver and initiate connection to my remote DB I get the following Pop up (twice) and it doesn't matter if I press Cancel or OK I still manage to connect to my DB successfully. I tried to do the same thing with another DB management tool called TablePlus and it does not go into any issues with the same settings. Initially I thought that the reason for this problem can be that the Pop up is asking for SSL passphrase/password which I did not set.
And also it does not make any change to it if I tick "Save password".

<img width="540" alt="Screenshot 2021-07-20 at 22 56 03" src="https://user-images.githubusercontent.com/2620949/126459398-299d1609-5aa9-4efe-a216-c8a7a6ca41bd.png">

It does not make a difference when I tick option to save password in the connection options either.

<img width="497" alt="Screenshot 2021-07-21 at 11 36 39" src="https://user-images.githubusercontent.com/2620949/126459545-5b5301c9-24a0-401e-917b-2652345592e9.png">

Tried to change Driver options to "No authentication" as well but with no results.

**Comment by uslss** [comment]
@PovilasC do you use SSH?
If yes, please check Save password option at SSH properties
![image](https://user-images.githubusercontent.com/31996417/127309134-92c20db1-90ec-470c-a34e-d3da66421a03.png)

**Reply by MiheevN** [reply]
No, SSH is not used. The USE SSH Tunnel checkbox is not checked.

What to do then?
